utils.py

```diff
@@ -9,3 +12,28 @@ def make_frame(ncols, nkey_cols, nrows, low=0, high=100):
     }
     val_columns = {f"val{i}": cp.random.rand(nrows) for i in range(nval_cols)}
     return cudf.DataFrame({**key_columns, **val_columns})
+
+
+def make_col(nrows, has_nulls=True):
+    c = cudf.core.column.as_column(cp.random.randn(nrows))
+    if has_nulls:
+        c[::2] = None
```

Comment:
Add a comment that this choice of where to put nulls is arbitrary in case we choose to make this more systematic or configurable in the future.